### ISSUE TITLE
SshNet.Security.Cryptography - The located assembly's manifest definition does not match the assembly reference

### ISSUE THREAD

**Issue by Compufreak345** [sshnet/SSH.NET#172]
Hi,

First of all : The package from Nuget (2016.0.0) works fine - but I am now trying to make some changes and am failing to build it properly. 

I'm quite sure I am doing something wrong, as this is my first .NET Core-project - I forked SSH.NET and tried to build the .NET Core-Part with my Visual Studio 2017 Enterprise Version 15.0.26206.0 D15REL - but I always get the following error when I use the library (either by referencing a published dll or the project directly). 

This is what happens when I try to initialise an SSH-Client in my .NET Core 1.1 (and 1.0) -App : 

> System.IO.FileLoadException: Could not load file or assembly 'SshNet.Security.Cryptography, Version=1.2.0.0, Culture=neutral, PublicKeyToken=2fa9220ff3eadda4' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)

Maybe you can give me a hint what I am doing wrong. I tried with your current develop-branch as well as a branch I forked from the 2016.0.0-release (that's what I want to base my changes on). I also tried removing the Nuget-Package for SshNet.Security.Cryptography and adding it again, but it didn't change the behavior.

Thanks & best regards,
Christoph

**Comment by drieseng** [comment]
The **Renci.SshNet.NETCore** project doesn't even reference **SshNet.Security.Cryptography**.
I propose using the VS 2017 solution in the **develop** branch.

**Comment by Compufreak345** [comment]
Are you sure? The Vs 2017 solution references ( https://github.com/sshnet/SSH.NET/blob/develop/src/Renci.SshNet.VS2017.sln#L28 ) the Renci.SshNet.NETCore\Renci.SshNet.NETCore.csproj which references SshNet.Security.Cryptography at https://github.com/sshnet/SSH.NET/blob/develop/src/Renci.SshNet.NETCore/Renci.SshNet.NETCore.csproj#L35 .

**Comment by Compufreak345** [comment]
Thanks for your help - I got it to run for me, but it still seems a bit odd. I took the develop VS2017-solution as a base. 

It still seems to me that for .NETStandard 1.3 the Renci.SshNet.NETCore references SshNet.Security.Cryptography ( https://github.com/sshnet/SSH.NET/blob/develop/src/Renci.SshNet.NETCore/Renci.SshNet.NETCore.csproj#L35 ) - but if I try to build the solution for a higher .NETStandard I get errors like 

> 'Connecting to a remote endpoint is not implemented.'	Renci.SshNet.NETCore	Renci.SshNet\Abstractions\SocketAbstraction.cs	114

>  'Receiving data from a Socket is not implemented.'	Renci.SshNet.NETCore	Renci.SshNet\Abstractions\SocketAbstraction.cs	183	Active

> 'Sending data to a Socket is not implemented.'	Renci.SshNet.NETCore	Renci.SshNet\Abstractions\SocketAbstraction.cs	440	Active

However, I was able to add the project to a simple .NET Core 1.1 console project now and it seems to work - but I was still having the same issues using it in a solution that is built against .NET 4.6.2, using .NET Core (I need this for implementing a service...).  I solved this by using the normal Renci.SSHNet - project and building it against .NET 4.6.2. 

This version-stuff keeps confusing me heavily, sorry for this :> Closing this issue as I found a working configuration.
